lib/linkerd2-proxy-resolve/src/destination/resolution.rs

```diff
@@ -165,10 +166,11 @@ where
                 .expect("resolution should not have been dropped");
 
             let query = self.query.take().expect("invalid state");
-            let ctx = logging::Section::Proxy.bg(LogCtx(query.authority().clone()));
 
-            let daemon = ctx.future(Daemon { updater, query });
-            tokio::spawn(Box::new(daemon));
+            let authority = query.authority().clone();
+            let fut =
+                Daemon { updater, query }.instrument(info_span!("resolve", authority = %authority));
```

Comment:
Nit: this can actually just be
```suggestion
                Daemon { updater, query }.instrument(info_span!("resolve", %authority));
```
since `authority` is a variable in scope.

Also, `tracing` doesn't require the authority to be cloned, it can be recorded by reference

Reply:
but it's moved into `Daemon`, so it has to be